### ISSUE TITLE
France has a FIPS code in the daily report for 04-10-2020  ¯\_(ツ)_/¯

### ISSUE THREAD

**Issue by Lucas-Czarnecki** [CSSEGISandData/COVID-19#2116]
France, for some reason, has a FIPS code (i.e., 00250) in the daily report for April 10th, 2020. 

No idea why.

**Comment by Inserm-CRI-LBI** [comment]
Indeed, can confirm. Also the case for April 9th, 2020. In these lines, the last entries also have " character inserted, which makes it a pain to parse. 
Normal day example: 

,,,France,2020-04-08   22:51:39,46.2276,2.2137,112950,10869,21254,80827,France

Problematic day example: 

00250,,,France,2020-04-09   23:02:19,46.2276,2.2137,117749,12210,23206,82333,",,France"

I could write an exception for my parser but maybe the problem is merely for a couple of days so no biggie. 

Anyway thanks for the awesome source of information.

**Comment by Lucas-Czarnecki** [comment]
The problem with FIPS remains. Affected daily reports include: 

- 04-09-2020
- 04-10-2020
- 04-11-2020
